pkg/services/ngalert/ngalert.go

```diff
@@ -397,7 +397,7 @@ func configureHistorianBackend(cfg setting.UnifiedAlertingStateHistorySettings,
 			BasicAuthPassword: cfg.LokiBasicAuthPassword,
 			TenantID:          cfg.LokiTenantID,
 		})
-		if err := backend.TestConnection(); err != nil {
+		if err := backend.TestConnection(context.Background()); err != nil {
```

Comment:
You might want a timeout here, but I'm not sure what value you would like?

Reply:
Isn't there already a timeout at the client level?

Reply:
There is a default 30 seconds timeouts for `loki_http`, but I was thinking you might want a timeout here that is potentially shorter, or at least consistent across all pluggable backends.

Reply:
There is, `30s`.

Reply:
I would definitely recommend we use a timeout of some sort for the context because then cancelation can be used for any additional additional work don't by a pluggable backend, not just the HTTP request.

Reply:
I would definitely recommend we use a timeout of some sort for the context because then cancelation can be used for any additional additional work done by a pluggable backend, not just the HTTP request.